WTWSMS/decisions/VIET_culture_decisions.txt

```diff
@@ -198,15 +198,18 @@ decisions = {
 			age = 16
 			scaled_wealth = 0.5
 			prisoner = no
-			religion_group = christian
-			NOT = { has_character_flag = VIET_celtic_high_cross }
 			OR = {
-				culture = saxon
+				religion = insular
+				religion = pelagianism
 				AND = {
-					culture_group = celtic
-					NOT = { culture = breton }
+					religion_group = christian
+					OR = {
+						culture = saxon
+						culture = old_saxon
+					}
```

Comment:
You said Saxons, and the code said Saxons. I removed the Saxons, but added them back when you explained to me that Saxons all erected crosses.

I still believe that non-Christian Celtics and Saxons should use the Runestone mechanics instead of erecting a Christian cross. I mean, why would they erect a cross to a God they do not believe in? Were not the Celtic Pagans hostile to the Christian encroachment on their lands?